src/components/Table/Table.css

```diff
@@ -12,6 +12,13 @@
 
 .table thead.fixed {
   position: sticky;
+  top: 0;
+}
+
+/* When the panel-body is the first child of the panel,
```

Comment:
I was having a hard time figuring this change out, but I think I got there: this isn't active yet! Right? I don't think we have any panel-bodies with a `thead` but without a `panel-header`. Are you planning on adding some?

Reply:
Ah right. So, correct: we no longer have a situation where this is necessary. Prior to this commit, panel headers scrolled out of view, so this CSS was necessary; now, it’s only necessary when there is no panel header at all, since otherwise the header stays fixed.

I’d rather leave this in because it’s such an esoteric fix to a “supported by design” scenario — it’s perfectly fine to have header-less panels, even though we don’t currently.

Reply:
Sounds good! Just making sure I understood. :smiley:

Reply:
This prompted me to clarify the related comment. If you were confused, so will others / me-later-on. 🙏